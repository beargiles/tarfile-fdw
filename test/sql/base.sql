\set ECHO 0
BEGIN;
\i sql/tarfile-fdw.sql
\set ECHO all

-- You should write your tests

SELECT tarfile-fdw('test');

ROLLBACK;
