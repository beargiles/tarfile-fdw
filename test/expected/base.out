\set ECHO 0
-- You should write your tests
SELECT tarfile-fdw('test');
 tarfile-fdw 
-------------
 
(1 row)

ROLLBACK;
